astraconnector_operator.yaml

```diff
@@ -0,0 +1,496 @@
+apiVersion: v1
```

Comment:
Copied over this file from integration. 
Once this PR merges this file will be updated by cicd